src/scripts/react/components/Join.js

```diff
@@ -186,30 +181,32 @@ const Join = () => (
               <h3 className="join-rules__sub-title">НЕОБХІДНО МАТИ</h3>
               <ul className="join-rules__list">
                 <li className="join-rules__list-item">
-                  паспорт або свідоцтво <br />
-                  про народження <br />
-                  для дитини
+                  паспорт або свідоцтво про народження для дитини
                 </li>
                 <li className="join-rules__list-item">
-                  податковий номер &nbsp;
-                  <div
-                    className="tooltip join-rules__list-tooltip"
-                    data-message="за наявності"
-                  />
+                  <div>
+                    податковий номер&nbsp;
+                    <div
+                      className="tooltip join-rules__list-tooltip"
+                      data-message="за наявності"
+                    />
+                  </div>
                 </li>
                 <li className="join-rules__list-item">
-                  мобільний телефон &nbsp;
-                  <div
-                    className="tooltip join-rules__list-tooltip"
-                    data-message="за наявності"
-                  />
+                  <div>
+                    мобільний телефон&nbsp;
+                    <div
+                      className="tooltip join-rules__list-tooltip"
+                      data-message="за наявності"
+                    />
+                  </div>
                 </li>
               </ul>
               <h3 className="join-rules__sub-title">ДАЛІ</h3>
               <ul className="join-rules__next-list join-rules__next-list_three">
                 <li className="join-rules__next-list-item">
                   На сайті eHealth в розділі “Мапа” <br />
-                  ознайомитись з
+                  ознайомитись з{" "}
```

Comment:
What is it? O_o

Reply:
Sure

Reply:
> In cases where you want to preserve the space adjacent to a newline, you can write {'Monkeys: '} or Monkeys:{' '} in your JSX source.
https://reactjs.org/blog/2014/02/20/react-v0.9.html#jsx-whitespace

Reply:
&nbsp; ?

Reply:
Two words separated by a non-breaking space will stick together, this is not our desired behavior.